INSTALL.md

```diff
@@ -0,0 +1,31 @@
+# TRex install role
+The ansible install-trex role downloads, unarchives, and installs trex on your
+trafficgen host.
+
+## Configfure Ansible inventory file
+Add `trafficgen` host to /etc/ansible/hosts
```

Comment:
You really shouldn't have to do this.  Instead, one would use the `-i` option of `ansible-playbook` to specify an inventory where the playbook would apply the role.